templates/common/page_start.html

```diff
@@ -19,7 +19,7 @@
     <title>Weasyl</title>
 
   <link rel="stylesheet" href="${CDNIFY('/static/fonts/museo500.css')}" />
-  <link rel="stylesheet" href="${CDNIFY('/static/public.css?' + SHA)}" />
+  <link rel="stylesheet" href="${resource_path('css/site.css')}" />
```

Comment:
Should we still append the '?SHA'? The reason we do it is to make the css CDNable.